### ISSUE TITLE
Move to new Ubuntu version

### ISSUE THREAD

**Issue by OgreTransporter** [bigbluebutton/bigbluebutton#9243]
**Is your feature request related to a problem? Please describe.**
After the release cycle of Ubuntu (https://ubuntu.com/about/release-cycle) there will be no more free updates for Ubuntu 16.04 next year.

**Describe the solution you'd like**
Wouldn't it make sense to switch to Ubuntu 18.04 (2023) or Ubuntu 20.04 (2025)? Maybe you can offer installation instructions and packages for all active LTS releases. The version for the oldest LTS is of course the most stable, but the users have the possibility to test the system with new LTS versions. This also increases the software quality, because if an LTS - as from 21.04 - does not get any more updates, there are tested packages for the next version.

**Comment by Sea444** [comment]
see #8474

**Comment by OgreTransporter** [comment]
I stand by my proposal to make BBB available in parallel for all active LTS in future. Yes, this will complicate the management of dependencies. But you have more time for the development of new releases, because then there will certainly be a release that runs on an active LTS.

**Comment by anarcat** [comment]
i think this request should be closed as a duplicate of #8474

**Comment by ffdixon** [comment]
duplicate of #8474

**Comment by npettiaux** [comment]
With BBB in Docker is it possible to install it on a Ubuntu 20.04 that would be on the metal ? I have a brand new server, bought to run BBB, but today I would rather install ubuntu 20.04 than 16.04. Much thanks of your input

**Comment by OgreTransporter** [comment]
@all Some of the dependencies are very outdated and don't get patched anymore - and not just since January.
```
================================================================================
================================================================================

                              DEPRECATION WARNING

  Node.js 8.x LTS Carbon is no longer actively supported!

  You will not receive security or critical stability updates for this version.

  You should migrate to a supported version of Node.js as soon as possible.
  Use the installation script that corresponds to the version of Node.js you
  wish to install. e.g.

   * https://deb.nodesource.com/setup_10.x — Node.js 10 LTS "Dubnium" (recommended)
   * https://deb.nodesource.com/setup_12.x — Node.js 12 LTS "Erbium"

  Please see https://github.com/nodejs/Release for details about which
  version may be appropriate for you.

  The NodeSource Node.js distributions repository contains
  information both about supported versions of Node.js and supported Linux
  distributions. To learn more about usage, see the repository:
    https://github.com/nodesource/distributions

================================================================================
================================================================================
```

@npettiaux  That sounds very interesting. I'm afraid I don't know much about Docker. Could you maybe write a manual on how to install it in Docker?

**Comment by npettiaux** [comment]
so, if I understand well, it is wise for BBB to upgrade soon to another supported version of ubuntu. I am not a specialist of containers like lxc or vservers or Docker. I hwva tried a little. What I know is that such tools allow for the installation of many tools in different environnement, with a rather strong separation, but sharing the same kernel, which means that the overhead is much reduced with respect to any other full virtualization solution like qemu, kvm, vmware, virtualbox. I would expect that BBB in a container would be nearly as good as directly on bare metal, while also much simpler to deploy, maintain, upgrade ...

**Comment by alangecker** [comment]
@OgreTransporter 
> Could you maybe write a manual on how to install it in Docker?

just follow the README here https://github.com/alangecker/bigbluebutton-docker ;)
